daemon/src/model/cfd.rs

```diff
@@ -655,6 +648,14 @@ impl Cfd {
         ))
     }
 
+    pub fn handle_rollover_accepted_taker(&self) -> Result<Event> {
```

Comment:
I chose to add this for two reasons:

1. I want to be able to emit the accepted event on the taker side, so that we can update de UI.
2. I want to ensure that we only proceed with the rollover if the CFD's state still agrees with it.

Reply:
Notes: 
ad. (1): This is actually more than we did before - the taker was not informed about accept before
ad. (2): Makes sense, but the time between sending the proposal and it being accepted should (usually) be within seconds. 

I think it is nice adding this step, but I don't think it is strictly necessary. Yes, going on with rollover if the Cfd is already final might cause weird side effects. I don't think it is such a big deal though, because once we apply the rollover we check again, so it should fail then. This *is* of course dangerous, because we already exchnaged parameters with the other party then, and the other party might not have noticed finality. In the end the `Cfd` would always be final though (once lock is spent it is spent...).

That said: happy to keep it, not sure if the state "helps" the user's understanding though. But that can be fixed by not showing fine-granular states in the UI.
If we want a separate state in the UI we might have to extend the state mapping in the projection actor.

Reply:
> ad. (2): Makes sense, but the time between sending the proposal and it being accepted should (usually) be within seconds.

That is true, but this snippet should allow the user to know what's going after the other party _accepts_. It should transition to `ContractSetup` (we could eventually introduce a dedicated state). Otherwise the UI would still show `Sent` or whatever we're doing atm.

From your comments I think we're okay to keep this, but thanks for thinking it through.